README.md

```diff
@@ -26,6 +29,22 @@ __Unfortunately, there is no support for handling a CAPTCHA, if the response con
 
 If you notice that for some reason cloudscraper stopped to work, do not hesitate and get in touch with me ( by creating an issue here, for example), so i can update it.
 
+Migration from v2 to v3
```

Comment:
@pro-src I hope I didn't miss any other crucial points?

Reply:
@codemanki LGTM

Reply:
We might want to advice proper usage of `error.error` which can be an `instanceof Error` in some cases. `request-promise` defines `error.error` but I believe it is a legacy attribute. I think we should promote `error.cause` since we're not setting those attributes ourselves. It's `request/promise-core`'s error library that is being used to set those attributes.

Reply:
@codemanki, The [![Build Status](https://travis-ci.org/codemanki/cloudscraper.svg?branch=master)](https://travis-ci.org/codemanki/cloudscraper) badge at the very top of this README.md should be removed. It's a duplicate.